weave/flow/eval_imperative.py

```diff
@@ -323,6 +331,16 @@ class EvaluationLogger(BaseModel):
     _evaluate_call: Call | None = PrivateAttr(None)
     _pseudo_evaluation: Evaluation = PrivateAttr()
 
+    # These properties forward useful properties of the Evaluate call onto
```

Comment:
A little confused by plural properties, is this referring to something besides `ui_url`?

Reply:
It originally included another property, but I removed it.

I'll rename